pkg/osvscanner/osvscanner.go

```diff
@@ -969,3 +971,23 @@ func makeLicensesRequests(packages []scannedPackage) ([][]models.License, error)
 
 	return licenses, nil
 }
+
+func overrideGoVersion(r reporter.Reporter, packages []scannedPackage, configManager *config.ConfigManager) {
+	// Overrides Go version
+
+	if len(packages) == 0 {
+		return
+	}
+
+	configToUse := configManager.Get(r, packages[0].Source.Path)
```

Comment:
Let's have this only get the config when we find stdlib. The config that's being loaded is already being cached, we can just move this Get call into the for loop (line 989)